apps/music/test/unit/remote_controls_test.js

```diff
@@ -26,6 +33,7 @@ suite('Media Remote Controls', function() {
       mrc._commandHandler(AVRCP.PLAY_PRESS);
       assert.ok(playListener.calledOnce);
     });
+
```

Comment:
I'm not sure it's needed to create one space line for each test cases. Since @dominickuo is the author, let him comment here for the coding style.